docs/pages/database-access/reference/aws.mdx

```diff
@@ -3,69 +3,75 @@ title: Database Access AWS IAM Reference
 description: AWS IAM policies for Teleport database access.
 ---
 
-## Auto-discovery
-With the appropriate IAM permissions, Teleport automatically discovers and
-configures IAM policies for Amazon RDS and Redshift. Teleport also requires
-permission to update database configurations, for example, to enable IAM
-authentication on RDS databases.
-
-For Amazon ElastiCache and MemoryDB, Teleport requires permission to
-automatically discover the Redis clusters. Teleport also requires permission to
-automatically discover and modify any Teleport-managed ElastiCache or MemoryDB
-users and permission to manage the passwords in AWS Secrets Manager.
-
-You can generate and manage the permissions with the [`teleport db configure
-bootstrap`](../../database-access/reference/cli.mdx#teleport-db-configure-bootstrap)
+Teleport Database Service requires IAM permissions for various tasks depending
+on the database type and setup, such as discovering endpoints and metadata of
+the database servers, generating IAM authentication tokens, and assuming IAM
+roles.
+
+You can generate the IAM permissions with the [`teleport db configure aws
```

Comment:
```suggestion
You can generate IAM permissions with the [`teleport db configure aws
```
Since the IAM permissions we're referring to here aren't specific ones.